workload.go

```diff
@@ -70,53 +63,6 @@ func NewWorkloadConfigWithDefaults(c WorkloadConfig) WorkloadConfig {
 		panic("must specify workload Name")
```

Comment:
Were you in the middle of removing `NewWorkloadConfigWithDefaults`?  It's still used in a couple of places, but there doesn't seem to be any particular reason.  And this `Name` check could go into `NewWorkload`.

Reply:
Good catch. I missed it in all the chaos.